### PR TITLE
fix(lite): add get-model docs

### PR DESCRIPTION
**Pull request by chenqy4933**
添加了 MegEngine 获得 lite 模型的文档，以及 python lite中使用 TensorBatchCollector 的demo。

@xxr3376 @MegChai please review!

### CODE REVIEW

File: source/user-guide/deployment/lite/quick-start/get-model.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+.. _get-model:
+
+===================================
+获得 MegEngine Lite 推理的模型
+===================================
+MegEngine 训练时候使用的动态图进行训练，当模型训练完成之后，需要将动态图转化为静态图，才能在 MegEngine Lite 中进行推理。目前有两种方法可以从
+训练的模型转换到推理模型的路径有以下两种方式：
```
Comment (MegChai) [inline source/user-guide/deployment/lite/quick-start/get-model.rst:7]: > 目前有两种方法可以从训练的模型转换到推理模型的路径有以下两种方式：

这句话有语病，望修改。

File: source/user-guide/deployment/lite/interface/python.rst [hunk 2]
```diff
@@ -936,6 +940,8 @@ set_log_level
 
 设置 MegEngine Lite 的 log 级别，改函数不在 LiteGlobal 类中，是一个独立的全局函数。
 
+.. _utils_api:
```
Comment (MegChai) [inline source/user-guide/deployment/lite/interface/python.rst:943]: 这里最好也加一个 lite, 无法保证其它地方不会出现 utils API

File: source/user-guide/deployment/lite/quick-start/get-model.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+* 使用 trace_module 方式：通过 MegEngine 的 :ref:`traced_module-guide` 将动态图转换为 traced_module IR，在这个 IR 基础上可以进行图手术等，参考 :ref:`graphsurgeon-example`，最后再转化为可以在 MegEngine Lite 上运行的静态图模型。
```
Comment (MegChai) [inline source/user-guide/deployment/lite/quick-start/get-model.rst:9]:
```suggestion
* 使用 trace_module 方式：通过 MegEngine 的 :ref:`Traced Module <traced_module-guide>` 将动态图转换为 traced_module IR，在这个 IR 基础上可以进行图手术等，参考 :ref:`graphsurgeon-example`，最后再转化为可以在 MegEngine Lite 上运行的静态图模型。
```
:ref: 语法可以指定展示的 label name.

File: source/user-guide/deployment/lite/index.rst [hunk 4]
```diff
@@ -43,5 +43,6 @@ MegEngine 支持多种主流深度学习推理平台，包括 Arm，X86，Cuda
 .. toctree::
    :maxdepth: 1
 
+   quick-start/get-model
```
Comment (MegChai) [inline source/user-guide/deployment/lite/index.rst:46]: 这个 `get-model` 文件的路径应该是 `/lite/get-model`, 它是 lite 介绍的一部分，但不是 quick start 的一部分。

```suggestion
   get-model
```

对应地，需要挪动一下文件的位置。

File: source/user-guide/deployment/lite/quick-start/get-model.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+   @jit.trace(symbolic=True, capture_as_const=True)
+   def fun(data, *, net):
+      pred = net(data)
+      pred_normalized = F.softmax(pred)
+      return pred_normalized
+
+   fun(data, net=traced_resnet)
+   fun.dump("resnet18.mge", arg_names=["data"])
+
+上面代码完成了如下：
```
Comment (MegChai) [inline source/user-guide/deployment/lite/quick-start/get-model.rst:57]:
```suggestion
上面代码完成了如下步骤：
```

File: source/user-guide/deployment/lite/quick-start/get-model.rst [hunk 6]
```diff
@@ -0,0 +1,102 @@
+.. _get-model:
+
+===================================
+获得 MegEngine Lite 推理的模型
```
Comment (MegChai) [inline source/user-guide/deployment/lite/quick-start/get-model.rst:4]: 获取 **用于** MegEngine Lite 推理的模型

**Review by MegChai — CHANGES_REQUESTED** [review]
文中出现代码变量的地方，需要包起来进行显式的告知，比如

```
``traced_resnet``
```
